src/components/translations/filter-expert-fr.ts

```diff
@@ -177,6 +177,8 @@ const filterExpertFr = {
     shuntSusceptance1: 'Susceptance parallèle 1 (μS)',
     shuntConductance2: 'Conductance parallèle 2 (μS)',
     shuntSusceptance2: 'Susceptance parallèle 2 (μS)',
+    pairingKey: "clé d'appareillement",
```

Comment:
```suggestion
    pairingKey: 'Xnode UCTE',
```

Reply:
are you sure ? because it is the deprecated name

Reply:
See with Christelle..